light-client/src/fork_detector.rs

```diff
@@ -34,12 +34,12 @@ pub enum Fork {
 
 /// Interface for a fork detector
 pub trait ForkDetector: Send {
-    /// Detect forks using the given light block, trusted state,
+    /// Detect forks using the given the verified block, trusted block,
```

Comment:
Why the extra "the"?
```suggestion
    /// Detect forks using the given verified block, trusted block,
```